.github/workflows/ci-actions-incremental.yml

```diff
@@ -129,17 +135,6 @@ jobs:
             git --no-pager diff '*pom.xml' 1>&2
             exit 1
           fi
-      - name: Tar Maven Repo
-        shell: bash
-        run: tar -I 'pigz -9' -cf maven-repo.tgz -C ~ .m2/repository
```

Comment:
I'm not sure to understand why you remove this?

Reply:
It's not gone, I had to move it further down.